script/openqa

```diff
@@ -27,6 +27,8 @@ $ENV{'MOJO_LISTEN'} ||= 'http://localhost:9526/';
 # allow up to 20GB - hdd images
 $ENV{MOJO_MAX_MESSAGE_SIZE}   = 1024 * 1024 * 1024 * 20;
 $ENV{MOJO_INACTIVITY_TIMEOUT} = 300;
+# the default is EV, and this heavily screws with our children handling
+$ENV{MOJO_REACTOR} = 'Mojo::Reactor::Poll';
```

Comment:
I added it in all cases - we used Poll so far anyway. Otherwise we would have noticed ourselves that perl-EV breaks os-autoinst ;)